dm_cli/bin/dm

```diff
@@ -270,12 +275,13 @@ def delete_packages(context, data_source: str, path: str):
         executor.map(thread_function, packages)
 
 
-@cli.command(help="Initialize the data sources and import all packages.")
+@cli.command(help="Initialize the data sources and import all packages. (Only use this command if datasources do not exist in DMSS)")
 @click.argument("path", required=False, default=".")
 @click.pass_context
-def init(context, path: str):
+def init(context, path: str, fail_if_ds_exists=True):
```

Comment:
running dm init when data source already exists causes errors

Reply:
How does this fail?
I think we should look into fixing that issue instead of this workaround.